proposals/NNNN-high-level-interface-foundation.md

```diff
@@ -0,0 +1,421 @@
+# High level interface: Foundation
+
+* Proposal: [SDL-NNNN](NNNN-high-level-interface-foundation.md)
+* Author: [Kujtim Shala](https://github.com/kshala-ford)
+* Status: **Awaiting review**
+* Impacted Platforms: [ iOS ]
+
+## Introduction
+
+This proposal is about setting up a foundation to provide a high level developer interface to the iOS library. A separate proposal will address the high level Android interface. It proposes a solution to mimic the UI framework of the native OS SDKs. It contains an overview and basic design but won't go much into details of a specific section. Subsequent proposal will be created in the near future in order to provide detailed designs whenever necessary.
+
+As discussed in the steering committee meeting from March 20 (see [here](https://github.com/smartdevicelink/sdl_evolution/issues/379#issuecomment-374736496)) this proposal is a counterproposal to [0133 - Enhanced iOS Proxy Interface](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0133-EnhancediOSProxyInterface.md).
+
+## Motivation
+
+In order to work with SDL app developers need to learn a new API which mostly doesn't adopt familiar patterns. Including but not limited to 
```

Comment:
Please add a comma after "SDL" so that the sentence reads "In order to work with SDL, app developers need..."